package.json

```diff
@@ -72,8 +72,9 @@
     "husky": "3.1.0",
     "jest": "24.9.0",
     "nock": "11.7.0",
+    "npm": "6.13.4",
```

Comment:
[MISC] We hadn't npm at all in the first place ? If so, why do we need it now ?

Reply:
```
forest-express git:(chore/message-length) yarn why npm
yarn why v1.19.1
[1/4] 🤔  Why do we have the module "npm"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "npm@6.13.2"
info Reasons this module exists
   - "semantic-release#@semantic-release#npm" depends on it
   - Hoisted from "semantic-release#@semantic-release#npm#npm"
info Disk size without dependencies: "37.21MB"
info Disk size with unique dependencies: "53.14MB"
info Disk size with transitive dependencies: "69.51MB"
info Number of shared dependencies: 214
✨  Done in 1.13s.
```

Reply:
🙏